sql/entity_mapping.sql

```diff
@@ -1,16 +1,26 @@
-DELETE FROM entity_mapping WHERE uuid IN (
-    '710e53d4-806a-11e9-bc42-526af7764f64',
-    '710e58c0-806a-11e9-bc42-526af7764f64',
-    '710e5a32-806a-11e9-bc42-526af7764f64',
-    '710e5b5e-806a-11e9-bc42-526af7764f64',
-    '710e5f1e-806a-11e9-bc42-526af7764f64',
-    '710e6054-806a-11e9-bc42-526af7764f64',
-    '710e6298-806a-11e9-bc42-526af7764f64',
-    '710e66ee-806a-11e9-bc42-526af7764f64',
-    '710e6838-806a-11e9-bc42-526af7764f64',
-    '710e6964-806a-11e9-bc42-526af7764f64',
-    '710e6d1a-806a-11e9-bc42-526af7764f64',
-    '710e6e6e-806a-11e9-bc42-526af7764f64',
-    '710e6f9a-806a-11e9-bc42-526af7764f64',
-    '710e713e-806a-11e9-bc42-526af7764f64'
-);
\ No newline at end of file
+INSERT IGNORE INTO entity_mapping
+    (uuid,entity_mapping_type_id,entity1_uuid,entity2_uuid)
```

Comment:
@prashantsable1991 please add the column 'date_created', the value can be set to 'now()'